code/modules/clothing/under/miscellaneous.dm

```diff
@@ -88,43 +88,39 @@
 	icon_state = "solgovr"
 	item_color = "solgovr"
 
-/obj/item/clothing/under/rank/centcom_officer
-	desc = "It's a jumpsuit worn by CentComm Officers."
-	name = "\improper CentComm officer's jumpsuit"
-	icon_state = "officer"
-	item_state = "g_suit"
-	item_color = "officer"
-
-/obj/item/clothing/under/rank/centcom_commander
-	desc = "It's a jumpsuit worn by CentComm's highest-tier Commanders."
-	name = "\improper CentComm officer's jumpsuit"
-	icon_state = "centcom"
-	item_state = "dg_suit"
-	item_color = "centcom"
+/obj/item/clothing/under/rank/centcom/ranker

```

Comment:
/ranker? If you're going to change the path, maybe make it something clear, like "/officer" or "/navyofficer".